src/Resources/config/serialization/OLD/ViewedProductProperties.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" ?>
+<serializer xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns="http://symfony.com/schema/dic/serializer-mapping"
+            xsi:schemaLocation="http://symfony.com/schema/dic/serializer-mapping https://symfony.com/schema/dic/serializer-mapping/serializer-mapping-1.0.xsd"
+>
+    <class name="Setono\SyliusFacebookPlugin\DTO\Properties\ViewedProductProperties">
```

Comment:
Why do we have these files inside and `OLD` directory?

Reply:
Forgot to remove them. Thanks